src/Commands/CommandBus.php

```diff
@@ -205,13 +213,22 @@ protected function parseCommandsIn(Collection $message): Collection
      */
     protected function process($entity, Update $update)
     {
-        $command = $this->parseCommand(
+        [$command,$bot] = $this->parseCommand(
             $update->getMessage()->text,
             $entity['offset'],
             $entity['length']
         );
-
-        $this->execute($command, $update, $entity);
+        if ($this->isMyCommand($command, $bot)) {
+            $this->execute($command, $update, $entity);
+        }
+    }
+    
+    protected function isMyCommand($command, $bot) 
+    {
+        if ($this->me === null) {
+            $this->me =  $this->telegram->getMe();
+        }
+        return $bot === null || $this->me->username === $bot;
```

Comment:
I think instead of making a call to `getMe()` API (Which would be too many calls for every command), we could just use the username set in the config file itself.